app/audit/models/OptOutAuditModel.scala

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package audit.models
+
+import auth.MtdItUser
+import connectors.itsastatus.ITSAStatusUpdateConnectorModel.{ITSAStatusUpdateResponse, ITSAStatusUpdateResponseFailure, ITSAStatusUpdateResponseSuccess}
+import models.incomeSourceDetails.TaxYear
+import models.itsaStatus.ITSAStatus.ITSAStatus
+import play.api.libs.json.{JsObject, JsValue, Json, OFormat}
+import services.optout.OptOutProposition
+
+case class Outcome(
+                    isSuccessful: Boolean,
+                    failureCategory: Option[String],
+                    failureReason: Option[String]
+                  )
+
+
+object Outcome {
+  implicit val format: OFormat[Outcome] = Json.format[Outcome]
+}
+
+case class OptOutAuditModel(
+                             //                             mtdItUser: MtdItUser[_],
```

Comment:
Does MtdItUser[_] still need to be here? Or can it just be removed?

Reply:
it can likely go, thanks